content-src/components/DiscoveryStreamComponents/SafeAnchor/SafeAnchor.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+export class SafeAnchor extends React.PureComponent {
+  safeURI(url) {
+    if (!url) {
+      return "";
+    }
+    const {protocol} = new URL(url);
```

Comment:
If this doesn't contain `http`/`https` it will throw an error. We need to wrap it in a try catch.